resume/resume-2.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<title>Nick Long Resume</title>
+	</head>
+	<body>
+		<style>
+	body {
+		background-color:rgb(243, 240, 234);
+		font-family: monospace;
+	}
+	#contact {
+		text-align: center;
+	}
+	h2 {
+		font-style: italic;
+	}
+	.collapsible {
+		cursor: pointer;
+		width: 100%;
+		border: none;
+		text-align: left;
+		outline: none;
+	}
+	.active, .collapsible:hover {
+	}
+	.content {
+		padding: 0 5px;
+		max-height: 0;
+		overflow: hidden;
+		transition: max-height 0.1s ease-out;
+	}
+
+		</style>
+		<h1 id="contact">Nicholas C. Long<br>3222 NW 54th ST Apt B24<br>Seattle, WA 98107<br>614-325-2406<br><a href="mailto:logan.nichols@gmail.com">logan.nichols@gmail.com</a></h1>
+		<h2 class="collapsible">Professional Experience</h2>
+		<div class="content">
```

Comment:
using a `div` tag here as a generic way to format content that we want to be collapsed. more on div tags: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div
https://www.w3schools.com/tags/tag_div.ASP

also there's some different ways to do this...maybe I should have used `<ul id="job" class="content">`